libvast/src/detail/mmapbuf.cpp

```diff
@@ -131,7 +140,7 @@ bool mmapbuf::resize(size_t new_size) {
     // If the current mapping is a multiple of the page size, we can try (1).
     if (size_ % page_size() == 0) {
       auto flags = flags_ | MAP_FIXED;
-      auto map = mmap(map_ + size_, new_size - size_, prot_, flags, fd_, 0);
+      auto map = mmap(map_ + size_, new_size - size_, prot_, flags, fd_, offset_ + size_);
```

Comment:
This change seems likewise orthogonal to the Linux fix.

Also seems like we are exceeding 80 characters here; please wrap accordingly.

Reply:
> exceeding 80 characters

Checking basic coding style rules (80 characters, no unnecessary vertical whitespaces, 2-space indentation, ...) sounds like something we should automate.

Reply:
Absolutely. We're not at a point yet where we can run `clang-format` over every PR, but we should consider basic linting.